### PR TITLE
62 add event model and notification badge dropdown

### PR DESCRIPTION
**Pull request by tsyaeger**
This is a clean version of that which had already been pushed. It has been washed of stripe checkout files/refs.

### CODE REVIEW

File: client/src/api/users.js [hunk 1]
```diff
@@ -15,14 +15,15 @@ export const createOrLoginUser = async (method, userData) => {
     if (response.data.hasOwnProperty('err')) return { ...response.data };
 
     localStorage.setItem('jwtToken', response.data.token);
-
+    console.log(response.data.user.notificationCount)
```
Comment (evankp) [inline client/src/api/users.js]: Should probably get rid of any console logs before merging

File: client/src/dummyData/dropDownItems.js [hunk 2]
```diff
@@ -16,7 +16,20 @@ const locations = [
...
+        content: "blah blah"
+    },
+    {
+        content: "etc etc"
+    },
+    {
+        content: "yada yada"
+    }
+]
+
```
Comment (evankp) [inline client/src/dummyData/dropDownItems.js:30]: I know this is placeholder data, was this ticket to hook it up or not? looks ok anyhow

Comment (tsyaeger) [reply client/src/dummyData/dropDownItems.js]: No the hookup was not part of the request. And I was under the impression that the alerts would come from websockets -- is that your understanding as well?

File: client/src/api/users.js (hunk 1 above)
Comment (tsyaeger) [reply client/src/api/users.js]: thanks. just removed it.

File: client/src/dummyData/dropDownItems.js (hunk 2 above)
Comment (DevDelvo) [reply client/src/dummyData/dropDownItems.js:30]: We can probably handle hooking up message notifications in a later ticket

Comment (tsyaeger) [reply client/src/dummyData/dropDownItems.js]: yeah -- messages were to be hooked up afterwards. Let's talk about how to go about that.

**Pull request merged**